spec/inputs/eventlog_spec.rb

```diff
@@ -1,7 +1,11 @@
 require "logstash/devutils/rspec/spec_helper"
-require 'logstash/inputs/eventlog'
 
-describe LogStash::Inputs::EventLog, :windows => true do
+describe "LogStash::Inputs::EventLog", :windows => true do
+
+  before(:all) do
+    require 'logstash/inputs/eventlog'
```

Comment:
I think is worth opening an issue, so when we're back here we know we should address this kind of requirements, for now I can think about platform dependant plugins and the zeromq, but I guess there are others where the required libs are messing up.